Tests/test_Seq_objs.py

```diff
@@ -10,7 +10,7 @@
 import unittest
 import sys
 
-from Bio import BiopythonWarning
+from Bio import BiopythonWarning, SeqIO
```

Comment:
I'd prefer this import on a separate line.

Reply:
I was following the multi-import per line style the rest of the file uses, but sure, changed. :)

Reply:
I think they were all multiple imports (classes, functions, etc) from the same python file - but Biopython's historical mixed case module names makes this harder to see.